Viper/Viper/CountryList/View/CountryListViewController.swift

```diff
@@ -0,0 +1,91 @@
+// CountryListViewController.swift
+// Copyright © SolovevAA. All rights reserved.
+
+import UIKit
+
+/// Экран со списком фильмов
+final class CountryListViewController: UIViewController {
+    // MARK: - Constants
+
+    private enum Constants {
+        static let countryText = Text.countryTitle
+        static let countryListTableViewCellText = Text.countryListCellName
+    }
+
+    // MARK: - Public Properties
+
+    var presenter: CountryListViewOutputProtocol?
+
```

Comment:
```suggestion
    // MARK: - Public Properties

    var presenter: CountryListViewOutputProtocol?

```
ниже чем Visual properties должно находиться